### PR TITLE
Change unitRepresentType for standard units

### PR DESCRIPTION
**Pull request by truongdd03**
# Description

This PR changes the unitRepresentType of standard units from unused to raw (temperature units) and quantity (others). The changes from PR #747 also show up here so I mark it as a draft. If there is no issue, it can be merged after integrating PR #747.

## Type of change

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

### CODE REVIEW

File: src/server/models/Unit.js [hunk 1]
```diff
@@ -88,21 +88,21 @@ class Unit {
...
-			['Metric_ton', 'Metric ton', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
-			['Fahrenheit', 'Fahrenheit', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
-			['Celsius', 'Celsius', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false]
+			['kWh', 'kWh', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, true],
+			['MJ', 'MegaJoules', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ADMIN, false],
+			['BTU', 'BTU', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, true],
+			['M3_gas', 'cubic meters of gas', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
+			['kg', 'kg', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
+			['Metric_ton', 'Metric ton', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
+			['Fahrenheit', 'Fahrenheit', Unit.unitRepresentType.FLOW, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
```
Comment (huss) [inline src/server/models/Unit.js]: Temperature is raw in OED. The flow ones are for ga/hour, kW, etc.

**Review by huss — CHANGES_REQUESTED** [review]
I think this is okay after this one change. I think it should be double checked after the included PR changes are merged to be sure nothing slipped by my check.

File: src/server/models/Unit.js (hunk 1 above)
Comment (truongdd03) [reply src/server/models/Unit.js]: I have fixed this in the new commit.

**Review by huss — APPROVED** [review]

**Comment by huss** [comment]
Testing shows it works as expected. Tests failing until next PR that fixes testing. This is ready to merge. Thanks to @truongdd03 for noticing this and fixing it.

**Pull request merged**